consensus/src/consensus/mod.rs

```diff
@@ -581,6 +585,10 @@ impl ConsensusApi for Consensus {
         self.body_tips_store.read().get().unwrap().read().iter().copied().collect_vec()
     }
 
+    fn get_tips_len(&self) -> usize {
+        self.body_tips().len()
```

Comment:
please use `self.body_tips_store.read().get().unwrap().read().len()` to avoid the full clone of the set

Reply:
fixed in https://github.com/aspectron/rusty-kaspa/commit/1ee65c349a9e663878787973d73ae2d4b420f44d